src/Piling/PilingParameter.cs

```diff
@@ -12,31 +12,42 @@ internal class PilingParameter : ISharedParameter
         private ExternalDefinition _definition;
         private ISharedParameterManager _manager;
 
+        /// <inheritdoc/> 
         public string GroupName => "Piling";
+        /// <inheritdoc/> 
         public string Name { get; private set; }
+        /// <inheritdoc/> 
         public ParameterType Type { get; private set; }
+        /// <inheritdoc/> 
         public bool Editable { get; private set; }
+        /// <inheritdoc/> 
         public string Description { get; private set; }
+        /// <inheritdoc/> 
         public Guid Id { get; private set; }
         
         private PilingParameter(ISharedParameterManager manager)
         {
             _manager = manager ?? throw new ArgumentNullException(nameof(manager));
         }
 
-        public void SetParameterValue(Parameter parameter, double value)
+        /// <inheritdoc/> 
+        public bool SetParameterValue(Parameter parameter, double value)
         {
             if (parameter.Definition.Name.Equals(Name, StringComparison.CurrentCultureIgnoreCase))
             {
-                parameter.Set(value);
+                return parameter.Set(value);
             }
+
+            return false;
         }
 
+        /// <inheritdoc/> 
         public void Register(Application application)
         {
             _definition = (ExternalDefinition)_manager.RegisterParameter(application, this);
         }
 
+        /// <inheritdoc/> 
```

Comment:
My understanding is this is not part of c#/.net. I believe its a common convetion in other languages, and resharper supports it, but is this an approach we are comfortable with?

Reply:
Quick check in VS with Resharper disable, shows it support.

Standard approach to keeping comments DRY.